### PR TITLE
Replace `plugins/docker` with `techknowlogick/drone-docker`in ci

### PR DESCRIPTION
**Pull request by 6543**

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -717,7 +717,7 @@ steps:
 
   - name: publish
     pull: always
-    image: plugins/docker:linux-amd64
+    image: techknowlogick/drone-docker:linux-amd64
```
Comment (techknowlogick) [inline .drone.yml]: Can you update to use `:latest` tag? It is multi-arch so no need to target specific arch.

Comment (6543) [reply .drone.yml]: hehe sure

Comment (6543) [inline .drone.yml]:
```suggestion
    image: techknowlogick/drone-docker:latest
```

File: .drone.yml [hunk 2]
```diff
@@ -734,7 +734,7 @@ steps:
         - pull_request
 
   - name: publish-rootless
-    image: plugins/docker:linux-amd64
+    image: techknowlogick/drone-docker:linux-amd64
```
Comment (6543) [inline .drone.yml]:
```suggestion
    image: techknowlogick/drone-docker:latest
```

File: .drone.yml [hunk 3]
```diff
@@ -772,7 +772,7 @@ trigger:
 steps:
   - name: dryrun
     pull: always
-    image: plugins/docker:linux-arm64
+    image: techknowlogick/drone-docker:linux-arm64
```
Comment (6543) [inline .drone.yml]:
```suggestion
    image: techknowlogick/drone-docker:latest
```

File: .drone.yml [hunk 4]
```diff
@@ -814,7 +814,7 @@ steps:
 
   - name: publish
     pull: always
-    image: plugins/docker:linux-arm64
+    image: techknowlogick/drone-docker:linux-arm64
```
Comment (6543) [inline .drone.yml]:
```suggestion
    image: techknowlogick/drone-docker:latest
```

File: .drone.yml [hunk 5]
```diff
@@ -834,7 +834,7 @@ steps:
         - pull_request
 
   - name: publish-rootless
-    image: plugins/docker:linux-arm64
+    image: techknowlogick/drone-docker:linux-arm64
```
Comment (6543) [inline .drone.yml]:
```suggestion
    image: techknowlogick/drone-docker:latest
```

**Review by techknowlogick — APPROVED** [review]

**Pull request merged**